single_tech_samples/databricks/sample4_ci_cd/pyspark/devops/cd-pipeline.yml

```diff
@@ -0,0 +1,93 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger: none
+
+pr: none
+
+resources:
+  pipelines:
+  - pipeline: ci
+    source: mdw-dbx-pyspark-ci-pipeline
+
+stages:
+- stage: IT
+  displayName: Integration Test
+  variables:
+  - group: mdwdo-dbx-pyspark-test
+  - group: mdwdo-dbx-release-dev
```

Comment:
`mdwdo-dbx-release-dev` is not specified in the README. What variables should be defined here?

Reply:
Gonna to remove variable group _mdwdo-dbx-pyspark-test_, once Databricks cluster ID is also stored in provisioned key vaults instance, @meghapatilcode is helping on this now.